modules/programs/eza.nix

```diff
@@ -53,6 +53,9 @@ in
     };
 
     enableZshIntegration = lib.hm.shell.mkZshIntegrationOption { inherit config; };
+    enableXonshIntegration = lib.mkEnableOption "Xonsh integration" // {
+      default = true;
+    };
```

Comment:
This seems like a fairly niche shell, this shouldn't default to true. I've not had time to orient myself in this patchset, but perhaps it should done like `mkZshIntegrationOption`, `mkFishIntegrationOption`, etc, instead.